### PR TITLE
allow alb support for situations where waf is required

### PR DESCRIPTION
**Pull request by cdaniluk**

### CODE REVIEW

**Review by sblack4 — APPROVED** [review]
I see two lbs. A alb for https and a nlb for ssh

**Review by kmackowick — APPROVED** [review]

File: alb.tf [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  subnets         = var.alb_https_subnets
+  tags            = var.tags
+
+  dynamic "access_logs" {
+    for_each = var.access_logs_enabled ? ["this"] : []
+
+    content {
+      bucket  = var.access_logs_bucket
+      prefix  = var.access_logs_prefix
+      enabled = var.access_logs_enabled
```
Comment (smiller171) [inline alb.tf:59]: You're already only creating the block if this var is true, so there's no need to evaluate the truthiness again

```suggestion
      enabled = true
```

File: alb.tf [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+  tags        = var.tags
+  vpc_id      = var.vpc_id
+
+  health_check {
+    healthy_threshold   = 2
+    unhealthy_threshold = 2
+    timeout             = 3
+    path                = "/status"
+    interval            = 30
+  }
```
Comment (smiller171) [inline alb.tf:96]: As an Open Source module, the health check should probably be parameterized

**Review by smiller171 — APPROVED** [review]
PR achieves everything it sets out to do, but I feel the interface could be simplified significantly for greater ease in consuming the module.

**Pull request merged**